parmoo/surrogates/gaussian_proc.py

```diff
@@ -457,6 +442,8 @@ def __init__(self, m, lb, ub, hyperparams):
                    matches lb and ub. Each entry is a number (greater than 0)
                    specifying the design space tolerance for that variable.
                    By default, des_tols = [1.0e-8, ..., 1.0e-8].
```

Comment:
I believe this should be updated

Reply:
It's technically still true, but when used within ParMOO, a value is always passed for ``des_tol`` (determined by the design variable types and settings), so the default value should never be invoked unless you're using this class outside of ParMOO